docs/resources/User.md

```diff
@@ -1,8 +1,7 @@
 # Users Resource
 
 Users in Discord are generally considered the base entity. Users can spawn across the entire platform, be members of
-guilds, participate and text and voice chat, and much more. Users are separated by a distinction of "bot" vs "normal",
-although similar, bot users are automated users that are "owned" by other users. Unlike normal users, bot users do
+guilds, participate and text and voice chat, and much more. Users are separated by a distinction of "bot" vs "normal". Although they are similar, bot users are automated users that are "owned" by other users. Unlike normal users, bot users do
```

Comment:
Could you also change `participate and text and voice chat` to `participate in text and voice chat`